mesonbuild/interpreterbase.py

```diff
@@ -1192,6 +1192,12 @@ def string_method_call(self, obj: str, method_name: str, posargs: T.List[TYPE_nv
                     raise InterpreterException('substring() argument must be an int')
                 end = posargs[1]
             return obj[start:end]
+        elif method_name == 'replace':
+            if len(posargs) != 2:
+                raise InterpreterException('replace() takes exactly two arguments.')
```

Comment:
Oh wait, missing FeatureNew here.

Reply:
How do you add it when it's not a function decorator? I didn't know

Reply:
`FeatureNew.single_use()`

Reply:
Thanks guys. Got it in. Seeing if tests are green :green_circle: